diff --git a/node_modules/@expo/cli/build/src/start/server/AsyncNgrok.js b/node_modules/@expo/cli/build/src/start/server/AsyncNgrok.js
index 4c42029..8c0607f 100644
--- a/node_modules/@expo/cli/build/src/start/server/AsyncNgrok.js
+++ b/node_modules/@expo/cli/build/src/start/server/AsyncNgrok.js
@@ -93,7 +93,7 @@ function _interop_require_wildcard(obj, nodeInterop) {
 }
 const debug = require('debug')('expo:start:server:ngrok');
 const NGROK_CONFIG = {
-    authToken: '5W1bR67GNbWcXqmxZzBG1_56GezNeaX6sSRvn8npeQ8',
+    authToken: _env.env.EXPO_NGROK_AUTH_TOKEN || '5W1bR67GNbWcXqmxZzBG1_56GezNeaX6sSRvn8npeQ8',
     domain: 'exp.direct'
 };
 const TUNNEL_TIMEOUT = 10 * 1000;
@@ -218,15 +218,16 @@ class AsyncNgrok {
             return url;
         } catch (error) {
             const assertNgrok = ()=>{
+                const errorMessage = error != null && typeof error.toString === 'function' ? error.toString() : String(error);
                 if ((0, _NgrokResolver.isNgrokClientError)(error)) {
                     var _error_body_details;
                     throw new _errors.CommandError('NGROK_CONNECT', [
-                        error.body.msg,
-                        (_error_body_details = error.body.details) == null ? void 0 : _error_body_details.err,
+                        error.body?.msg,
+                        (_error_body_details = error.body?.details) == null ? void 0 : _error_body_details.err,
                         _chalk().default.gray('Check the Ngrok status page for outages: https://status.ngrok.com/')
                     ].filter(Boolean).join('\n\n'));
                 }
-                throw new _errors.CommandError('NGROK_CONNECT', error.toString() + _chalk().default.gray('\nCheck the Ngrok status page for outages: https://status.ngrok.com/'));
+                throw new _errors.CommandError('NGROK_CONNECT', errorMessage + _chalk().default.gray('\nCheck the Ngrok status page for outages: https://status.ngrok.com/'));
             };
             // Attempt to connect 3 times
             if (attempts >= 2) {
